package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Knock CLI",
   "author": "@knocklabs",
   "bin": {
-    "knock": "./bin/run"
+    "knock": "./bin/run.js"
```

Comment:
This bin script file used to be just `run` prior to v3 upgrade.

I tested by installing a local package with and without this change, and the bin executable is omitted without it.